mods/parsoid.js

```diff
@@ -14,13 +14,55 @@ var yaml = require('js-yaml');
 var fs = require('fs');
 var spec = yaml.safeLoad(fs.readFileSync(__dirname + '/parsoid.yaml'));
 
+// THIS IS EXPERIMENTAL AND ADDED FOR TESTING PURPOSE!
+// SHOULD BE REWRITTEN WHEN DEPENDENCY TRACKING SYSTEM IS IMPLEMENTED!
+var Purger = require('htcp-purge');
+var Template = require('../lib/reqTemplate');
+var cacheURIs = [
+    // /page/mobile-html/{title}
+    new Template({
+        uri: 'https://{domain}/api/rest_v1/page/mobile-html/{title}'
+    }),
+    // /page/mobile-html-sections/{title}
+    new Template({
+        uri: 'https://{domain}/api/rest_v1/page/mobile-html-sections/{title}'
+    }),
+    // /page/mobile-html-sections-lead/{title}
+    new Template({
+        uri: 'https://{domain}/api/rest_v1/page/mobile-html-sections-lead/{title}'
+    }),
+    // /page/mobile-html-sections-remaining/{title}
+    new Template({
+        uri: 'https://{domain}/api/rest_v1/page/mobile-html-sections-remaining/{title}'
+    }),
+    // /page/mobile-text/{title}
+    new Template({
+        uri: 'https://{domain}/api/rest_v1/page/mobile-text/{title}'
+    })
+
+];
 
 function ParsoidService(options) {
+    var self = this;
     options = options || {};
-    this.parsoidHost = options.parsoidHost
+    self.parsoidHost = options.parsoidHost
         || 'http://parsoid-lb.eqiad.wikimedia.org';
+
+    // THIS IS EXPERIMENTAL AND ADDED FOR TESTING PURPOSE!
+    // SHOULD BE REWRITTEN WHEN DEPENDENCY TRACKING SYSTEM IS IMPLEMENTED!
+    self.purgeOnUpdate = options.purge_on_update;
+    if (self.purgeOnUpdate) {
+        self.purger = new Purger({
+            routes: [
+                {
+                    host: '239.128.0.112',
```

Comment:
It would be nice to have the host and port configurable, so that we can test this in labs before going out with it. Have them as optional params to the parsoid module conf would be ideal